### PR TITLE
BCN-FT032020-Siiri S

### PR DESCRIPTION
**Pull request by fetaplop**
initialising

### CODE REVIEW

**Comment by fetaplop** [comment]
Done for today!

**Comment by fetaplop** [comment]
ughhh realised that the jasmine thingy worked if i commented out the test data so I did another pull request and I hope that didn't mess up everything! So Now I'm really done for today. :D

File: src/movies.js [hunk 1]
```diff
@@ -1,14 +1,151 @@
...
+//Create a ratesAverage() function that receives an array as a parameter and solves the challenge.
+//display it on the console. The rate must be returned rounded to 2 decimals!
+
+function ratesAverage(filmArr) {
+    const ratings = [];
+    filmArr.forEach( (film) => ratings.push(film.rate) ) // rip out all the ratings and put them in 1 array
+    //console.log(ratings);
+    avg = (ratings.reduce( (totalRatings, nextRating) => totalRatings + nextRating )) / (ratings.length); // sum all ratings with reduce() to calculate average
+    //console.log(avg); 
+    return avg;
```
Comment (tawebbcn) [inline src/movies.js:133]: You need to round the avg to 2 decimals and make sure it's a number, in order to pass all tests

File: src/movies.js [hunk 2]
```diff
@@ -1,14 +1,151 @@
...
+ratesAverage(movies);
+
 
 // Iteration 5: Drama movies - Get the average of Drama Movies
+function dramaMoviesRate(filmArr) {
+    // filter together drama films
+    const dramas = filmArr.filter( film => film.genre.includes("Drama")  ) // if film genre is drama, add it to dramas array
+    //console.log(dramas);
+    dramaAvg = ratesAverage(dramas);
+    //console.log("dramaAvg", dramaAvg); UNCOMMENT THIS TO GET AVERAGE CONSOLE LOGGED!
```
Comment (tawebbcn) [inline src/movies.js:145]: well done reusing the previous function 😉

**Pull request closed without merging**